src/flowMax.coffee

```diff
@@ -38,17 +40,20 @@ flowMax = (...funcs) ->
         isAddWrapperHOC(func) or
         isBranch func
       )
-        newFollowingFlowMax = flowMax ...getFollowingFuncs(funcIndex)
+        wrappedDisplayName = switch
+          when isAddPropTypes func
+            wrapExistingDisplayName 'addPropTypes'
+          when isAddWrapper func then wrapExistingDisplayName 'addWrapper'
+          when isAddWrapperHOC func
+            wrapExistingDisplayName 'addWrapperHOC'
+        newFollowingFlowMax = flowMax(
+          ...getFollowingFuncs(funcIndex, {wrappedDisplayName})
```

Comment:
I believe that the existing behavior was that a display name set via `addDisplayName()` would only propagate "one level down" (so eg a component with an `addDisplayName('Component')`, an `addWrapper()` and a `branch()` would result in components with display names `Component`, `addWrapper(Component)` and `branch()` (rather than `Component, `addWrapper(Component)` and `branch(addWrapper(Component))`

This was because the new "rest of the chain" component/`flowMax()` didn't have a display name set

So now manually prepend an `addDisplayName()` to the new `flowMax()` (this seems safe?)